lib/control/tests/test_control_cmds.c

```diff
@@ -163,8 +184,6 @@ Test(control_cmds, test_stats)
   gchar **stats_result;
   const gchar *response;
 
-  stats_init();
```

Comment:
If I understand correctly, you can remove this, because you added `app_startup` to setup, so it is not necessary to reinitialize.

With the same reasoning, can you remove the counterpart: `stats_destroy`? It is called in `app_shutdown`.

Reply:
Yes, I will remove ```stats_destroy```.